images/macos/tests/BasicTools.Tests.ps1

```diff
@@ -166,4 +166,30 @@ Describe "Homebrew" {
     It "Homebrew" {
         "brew --version" | Should -ReturnZeroExitCode
     }
+}
+
+Describe "Kotlin" {
+    It "kapt" {
+        "kapt -version" | Should -ReturnZeroExitCode
+    }
+
+    It "kotlin" {
+        "kotlin -version" | Should -ReturnZeroExitCode
+    }
+
+    It "kotlinc" {
+        "kotlinc -version" | Should -ReturnZeroExitCode
+    }
+
+    It "kotlinc-js" {
+        "kotlinc-js -version" | Should -ReturnZeroExitCode
+    }
+
+    It "kotlinc-jvm" {
+        "kotlinc-jvm -version" | Should -ReturnZeroExitCode
+    }
+
+    It "kotlin-dce-js" {
+        "kotlin-dce-js -version" | Should -ReturnZeroExitCode
+    }
```

Comment:
I just thought that we can simply retrieve all the binaries in kotlin bin directory and check them in a loop instead of hardcoding this list

Reply:
Or at least, we can use It loop here:
```
$tools = @("kapt", "kotlin", "kotlinc", "kotlinc-js", "kotlinc-jvm", "kotlin-dce-js")
It -TestCases ($tools | ForEach-Object { { Name = $_ } }) {
  "$Name -version" | Should -ReturnZeroExitCode
}
```